assets/cms/scss/panels/pages/details/_clipboard.scss

```diff
@@ -26,10 +26,17 @@
     .ccm-panel-add-block-draggable-block-type {
       background-color: white;
       border: 1px solid $gray-200;
+      cursor: grab;
       margin: 5px 30px;
       max-height: 120px;
       min-height: 120px;
       overflow: hidden;
+
+      .block-content {
+        margin-right: -280px;
```

Comment:
What's this right margin relative to? Should we also hardcode the width somewhere in here so that it doesn't get changed elsewhere accidentally?

Reply:
Is not relative to anything, I manually tested how much would I need so is able to fit the content into the whole container, but base in our earlier conversation, we probably will be replacing the entire preview with the one is done in the stacks panel.